deep-learning/src/test/scala/com/microsoft/azure/synapse/ml/cntk/ImageFeaturizerSuite.scala

```diff
@@ -4,80 +4,47 @@
 package com.microsoft.azure.synapse.ml.cntk
 
 import com.microsoft.azure.synapse.ml.Secrets
-import com.microsoft.azure.synapse.ml.build.BuildInfo
-import com.microsoft.azure.synapse.ml.core.env.FileUtilities
 import com.microsoft.azure.synapse.ml.core.test.fuzzing.{TestObject, TransformerFuzzing}
-import com.microsoft.azure.synapse.ml.downloader.{ModelDownloader, ModelSchema}
 import com.microsoft.azure.synapse.ml.image.ImageTestUtils
 import com.microsoft.azure.synapse.ml.io.IOImplicits._
 import com.microsoft.azure.synapse.ml.io.powerbi.PowerBIWriter
 import com.microsoft.azure.synapse.ml.io.split1.FileReaderUtils
-import com.microsoft.azure.synapse.ml.onnx.ONNXModel
-import com.microsoft.azure.synapse.ml.opencv.ImageTransformer
 import org.apache.spark.injections.UDFUtils
-import org.apache.spark.ml.{Pipeline, PipelineModel}
 import org.apache.spark.ml.linalg.DenseVector
 import org.apache.spark.ml.util.MLReadable
 import org.apache.spark.sql.DataFrame
 import org.apache.spark.sql.functions.col
-import org.apache.spark.sql.types.{FloatType, StringType}
+import org.apache.spark.sql.types.StringType
 
-import java.io.File
-import java.net.URI
+// TODO move all to onnx package after review
 
-trait TrainedCNTKModelUtils extends ImageTestUtils with FileReaderUtils {
-
-  lazy val modelDir = new File(filesRoot, "CNTKModel")
-  lazy val modelDownloader = new ModelDownloader(spark, modelDir.toURI)
-
-  lazy val resNetUri: URI = new File(modelDir, "ResNet50_ImageNet.model").toURI
-  lazy val resNet: ModelSchema = modelDownloader.downloadByName("ResNet50")
-
-  def resNetModel(): ImageFeaturizer = new ImageFeaturizer()
-    .setInputCol(inputCol)
-    .setOutputCol(outputCol)
-    .setModel(resNet)
-
-}
+trait TrainedONNXModelUtils extends ImageTestUtils {
+  override def beforeAll(): Unit = {
+    spark
+    super.beforeAll()
+  }
 
-trait TrainedONNXModelUtils {
-  lazy val resNetOnnxUrl = "https://mmlspark.blob.core.windows.net/datasets/ONNXModels/resnet50-v1-12-int8.onnx";
+  lazy val resNetOnnxUrl = "https://mmlspark.blob.core.windows.net/datasets/ONNXModels/resnet50-v1-12-int8.onnx"
   lazy val resNetOnnxPayload: Array[Byte] = {
```

Comment:
Please move this to using ONNXHub. Thanks!

Reply:
This is specifically testing that we can pass in a model as an array of bytes, which is still an option.  Everything else is using OnnxHub.